orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationAiModelParameters.java

```diff
@@ -0,0 +1,86 @@
+package com.sap.ai.sdk.orchestration;
+
+import java.util.Map;
+import javax.annotation.Nonnull;
+
+/** Helper interface to define typed parameters. */
+@FunctionalInterface
+public interface OrchestrationAiModelParameters {
+  /**
+   * Get the parameters.
+   *
+   * @return the parameters.
+   */
+  @Nonnull
+  Map<String, Object> getParams();
+
+  /** GPT model parameters. */
+  interface GPT extends OrchestrationAiModelParameters {
+    /**
+     * Create a new builder.
+     *
+     * @return the builder.
+     */
+    @Nonnull
+    static GPT.Builder0 params() {
+      return maxTokens ->
+          temperature ->
+              frequencyPenalty ->
+                  presencePenalty ->
+                      () ->
+                          Map.of(
+                              "max_tokens", maxTokens,
```

Comment:
why this specific sub-selection of (Azure) OpenAI parameters?

Reply:
Only after asking Orchestration team, I realized they are supporting the parameters for non-OpenAI LLM providers as well. From the spec that wasn't apparent. As a result, we can change the solution to be model-agnostic.